README.md

```diff
@@ -733,8 +739,8 @@ Here's a basic performance benchmark on MacBook Pro Early 2013 run on Kubernetes
 
 ### golint
 
-```shell
-time golint kubernetes/... > /dev/null
+```zsh
```

Comment:
this is not a zsh, but a `console` or `shellsession`

Reply:
thank you